datamonster_api/lib/company.py

```diff
@@ -26,6 +26,9 @@ def __repr__(self):
         ticker_str = "[{}] ".format(self.ticker) if self.ticker else ""
         return "<{}: {}{}>".format(self.__class__.__name__, ticker_str, self.name)
 
+    def __hash__(self):
+        return int(self.id)
```

Comment:
i see the same changes (plus tests) in https://github.com/adaptivemgmt/datamonster-api/pull/31/files
do we want to combine them into one PR?

Reply:
it is trying to solve two separate issues - i think not

Reply:
i added this in here so that evan can do more testing

Reply:
the changes in that PR is literally a subset of this one (minus the tests)? if that's the case, i'd suggest carry over the tests here too since we did add the same changes